src/widget/woverview.cpp

```diff
@@ -58,30 +58,31 @@ WOverview::WOverview(
           m_pHoveredMark(nullptr),
           m_scaleFactor(1.0) {
     m_endOfTrackControl = make_parented<ControlProxy>(
-            m_group, QStringLiteral("end_of_track"), this, ControlFlag::NoAssertIfMissing);
+            m_group, QStringLiteral("end_of_track"), this);
     m_endOfTrackControl->connectValueChanged(this, &WOverview::onEndOfTrackChange);
     m_pRateRatioControl = make_parented<ControlProxy>(
-            m_group, QStringLiteral("rate_ratio"), this, ControlFlag::NoAssertIfMissing);
+            m_group, QStringLiteral("rate_ratio"), this);
     // Needed to recalculate range durations when rate slider is moved without the deck playing
     m_pRateRatioControl->connectValueChanged(
             this, &WOverview::onRateRatioChange);
-    m_trackSampleRateControl = make_parented<ControlProxy>(
-            m_group, QStringLiteral("track_samplerate"), this, ControlFlag::NoAssertIfMissing);
-    m_trackSamplesControl = make_parented<ControlProxy>(
-            m_group, QStringLiteral("track_samples"), this);
-    m_playpositionControl = make_parented<ControlProxy>(
-            m_group, QStringLiteral("playposition"), this, ControlFlag::NoAssertIfMissing);
+    // TODO(xxx) Use PollingControlProxy where possible
+    m_trackSampleRateControl = PollingControlProxy(
+            m_group, QStringLiteral("track_samplerate"));
+    m_trackSamplesControl = PollingControlProxy(
+            m_group, QStringLiteral("track_samples"));
+    m_playpositionControl = PollingControlProxy(
+            m_group, QStringLiteral("playposition"));
```

Comment:
Constructing PollingControlProxy like this throws a lot of
```
critical [Main] DEBUG ASSERT: "flags & ControlFlag::AllowMissingOrInvalid" in function PollingControlProxy::PollingControlProxy(const ConfigKey&, ControlFlags) at ./src/control/pollingcontrolproxy.h:24
warning [Main] ControlDoublePrivate::getControl returning nullptr for invalid ConfigKey "" , ""
critical [Main] DEBUG ASSERT: "!"Unexpected invalid key"" in function static QSharedPointer<ControlDoublePrivate> ControlDoublePrivate::getControl(const ConfigKey&, ControlFlags, ControlObject*, bool, bool, bool, double) at ./src/control/control.cpp:147
```
and I have no clue why.